src/commands/docs/utils.ts

```diff
@@ -0,0 +1,90 @@
+import { type CommandInteraction, MessageFlags } from 'discord.js';
+import { logToChannel } from '../../util/channel-logging.js';
+import type { ProviderConfig } from './types.js';
+
+export const SEARCH_TERM = '%SEARCH%';
+export const TERM = '%TERM%';
+
+// Utility functions
+export const getSearchUrl = (url: string, search: string) =>
+  url.replace(SEARCH_TERM, encodeURIComponent(search));
```

Comment:
I like it if is explicit with the curly braces and return statements.

```suggestion
export const getSearchUrl = (url: string, search: string) => {
  return url.replace(SEARCH_TERM, encodeURIComponent(search));
};
```